doc/design_paper.md

```diff
@@ -458,40 +458,48 @@ Any party is able to render and apply functions to the token using Tokenscript,
 **[[loudly thinking: Wouldn't it be better to continue with the payment side example? It somehow breaks the flow. Advantages doesn't seem to be a good fit in the chapter about requirements. Maybe I'll test it; maybe it's good to explain the advantages when we have also the payment side. The more I think about it, the more I tend to it. We already have a lot of talk about why Tokenscript. So we better stick to the examples and present the advantages later on - I tried it out.]]
```

Comment:
Can you put this in a comment:
method: https://stackoverflow.com/questions/4823468/comments-in-markdown